pkg/apis/discovery/validation/validation.go

```diff
@@ -38,10 +37,9 @@ var (
 		string(api.ProtocolUDP),
 		string(api.ProtocolSCTP),
 	)
-	maxTopologyLabels = 16
-	maxAddresses      = 100
-	maxPorts          = 100
-	maxEndpoints      = 1000
+	maxAddresses = 100
+	maxPorts     = 100
```

Comment:
this is going to collide with https://github.com/kubernetes/kubernetes/pull/99795 :)